src/events/components/SearchModule/search.less

```diff
@@ -1,4 +1,4 @@
-.dateInput {
+/* .dateInput {
```

Comment:
Why still have this large piece of code if it is not used?

Reply:
Me forgetting to remove it, and not wanting to delete the file, since it will still be used for the styling